ee/clickhouse/queries/test/test_paths.py

```diff
@@ -201,13 +208,34 @@ def _create_sample_data_multiple_dropoffs(self):
         for i in range(5):
             Person.objects.create(distinct_ids=[f"user_{i}"], team=self.team)
             _create_event(event="step one", distinct_id=f"user_{i}", team=self.team, timestamp="2021-05-01 00:00:00")
-            _create_event(event="step two", distinct_id=f"user_{i}", team=self.team, timestamp="2021-05-03 00:00:00")
-            _create_event(event="step three", distinct_id=f"user_{i}", team=self.team, timestamp="2021-05-05 00:00:00")
+            _create_event(
+                event="between_step_1", distinct_id=f"user_{i}", team=self.team, timestamp="2021-05-01 00:01:00"
```

Comment:
Minor nit: naming makes it seem like `between_step_2` happens between `step two` and `step three`, which made me take a back step when reading tests, thinking "this path response doesn't make sense"

Suggestion: Name these `between_step_1_a` , `between_step_1_b`, and `between_step_1_c`